docs/developinstallwin.rst

```diff
@@ -0,0 +1,147 @@
+Next, create and activate a new environment, which will contain your QGIS installation. The name of the environment
+can be anything, but since we will install the latest version of QGIS, we will call it *qgis_latest* in this example.
+Open your conda terminal and use::
+
+  conda create -n qgis_latest
+  conda activate qgis_latest
+
+To **install the latest QGIS version**, use::
+
+  conda install qgis --channel conda-forge
+
+.. Note::
+  Because conda will always try to install the latest version, if you want to use another version, you need to specify it, e.g.:
+
+  ``conda install qgis=3.34.11 --channel conda-forge``
+
+Now install avaframe into the qgis environment. This is to make sure that the plugin will find this version first, and install the necessary dependencies.
+In the conda terminal, use::
+
+  pip install avaframe
+
+Now find the directory where your environment is located. The path should be something like:
+``C:\Users\USER\miniconda3\envs\qgis_latest``. Change into it::
+
+  cd \miniconda3\envs\qgis_latest
+
+.. Note::
+  **Tip:** To find the path, you can do ``pip show avaframe`` and copy the
+  first part which shows your environment directory
+
+Clone the AvaFrame repository and change into it::
+
+  git clone https://github.com/avaframe/AvaFrame.git
+  cd AvaFrame
+
+.. Note::
+  This will pull the latest developer version of AvaFrame, if you want the current release version for the plugin,
+  you need to specify the version, e.g.:
+
+  ``https://github.com/avaframe/AvaFrame/tree/1.8.3``
+
+Compile the cython com1DFA part and create an editable avaframe version in your environment::
+
+  python setup.py build_ext --inplace
+  pip install -e .
+
+To **run QGIS**, in the terminal, activate the environment (if not activated already) and run the ``qgis`` command::
+
+  conda activate qgis_latest
+  qgis
+
+Now **install the AvaFrameConnector plugin** via QGIS as per usual (:ref:`installation:Operational Installation`).
+
+AvaFrame should now be installed, and both installations should work and be editable separately. To display the installed version and installation location, use
+``pip show avaframe`` in the respective environment, or use the *getVersion* function in the plugin. Note that the script
+versions are separate, and changes in one installation will not affect the other.
+
+**Updating QGIS**
+
+To update QGIS to the most recent version, you need to run the following command with the respective environment active::
+
+  conda update qgis -c conda-forge
+
+**Final comments:**
+
+Be aware that the QGIS Conda package does not provide the same user experience as the QGIS installer version.
+For example, there are no desktop icons or file association, and it does not include GRASS and SAGA, etc..
+
+On the other hand, the QGIS installation through Conda will share user configurations, installed plugins,
+with any other QGIS installation on your system.
```

Comment:
I would leave this out, since I'm not sure that is true for all setups...

Reply:
This info is from [here](https://plugins.qgis.org/planet/tag/conda/). True, could leave this out though, as I am not completely sure either how QGIS works there..

Reply:
The info you linked is from 2019 and as you/we are not completely sure how this is handled on current version, please just leave it out...